### ISSUE TITLE
please set "maintainer" for Dockerfile

### ISSUE THREAD

**Issue by n0mer** [cachethq/Cachet#872]
so header for `Dockerfile` will look like

```bash
FROM debian:jessie
MAINTAINER First Last &lt;me@doma.in&gt;
```

**Comment by n0mer** [comment]
of course, modify by assigning appropriate first/last and e-mail :)

**Comment by jbrooksuk** [comment]
What is this for exactly? As I mentioned in another issue, we're planning on moving Docker and Vagrant wrappers to their own repos soon, so it may be more apt then.

**Comment by n0mer** [comment]
This is `good citizen` behaviour in Docker world.

Imagine you have linux package w/out contact of it's maintainer

**Comment by jbrooksuk** [comment]
This will be more applicable for the new docker-specific repo we have.

**Comment by jbrooksuk** [comment]
Closed in 3f2f5cd25b2a7d48367cf0efcf05afbe58c38bd1
